docs/getting-started/security-ui.asciidoc

```diff
@@ -32,6 +32,20 @@ The navigation menu contains direct links and expandable groups, identified by t
 [role="screenshot"]
 image::images/nav-overview.gif[Overview of the navigation menu]
 
+[[visualization-actions]]
+[float]
+== Visualization actions
+
+Many {elastic-sec} histograms, graphs, and tables display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) when you hover over them, so you can examine the {es} queries used to retrieve data throughout the app. 
+
+[role="screenshot"]
+image::images/inspect-icon-context.png[Inspect icon,40%]
+
+Other visualizations display an options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]), allowing you to inspect the visualization's queries, add it to a new or existing case, or open it in Lens for customization.
```

Comment:
```suggestion
Other visualizations display an options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]) which allows you to inspect the visualization's queries, add it to a new or existing case, or open it in Lens for customization.
```
can't say why but the "ing" construction seems off to me

Reply:
```suggestion
Other visualizations display an options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]), which allows you to inspect the visualization's queries, add it to a new or existing case, or open it in Lens for customization.
```